modules/cnv-updating-imported-vm-network-name.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * modules/cnv-importing-vmware-vm.adoc
+[id="cnv-updating-imported-vmware-vm-network-name_{context}"]
+= Updating the imported virtual machine's NIC name
+
+Update the imported virtual machine's NIC name to conform to {ProductName} naming conventions.
+
+. Log in to the imported virtual machine.
+. Go to the */etc/sysconfig/network-scripts* directory.
+. Rename the network configuration file with the new NIC name (a single NIC is called `eth0`):
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+$ mv _original_NIC_ ifcfg-eth__x__
```

Comment:
Is this "x" supposed to be a replaceable value/number? If so, you might want to make that more apparent. Without seeing the source here, I wouldn't have been able to tell that it was italicized in the built docs.

I'm not sure how it would look here, but we typically do replaceable values as <replaceable_value_name>. I don't know what the appropriate description to use here is, but whatever it is, you could do ```$ mv _original_NIC_ ifcfg-eth<replaceable_value_name>```. And you could also add a callout here to explain, and perhaps add your note above about a single NIC being called `eth0`.

Reply:
`x` is a replaceable value here. I do not think this requires a callout or `<replaceable_value_name>` because the value is obvious to an admin. It's going to be eth0 or eth1 or eth2, etc., depending on how many NICs the VM has. This is a standard network device naming convention.